phoenix-core/src/main/java/org/apache/phoenix/util/IndexUtil.java

```diff
@@ -978,4 +981,16 @@ public static void setScanAttributesForIndexReadRepair(Scan scan, PTable table,
         }
         addEmptyColumnToScan(scan, emptyCF, emptyCQ);
     }
+
+    public static int getNumOfRegions(Configuration configuration, String tableName)
+            throws IOException {
+        int numRegions;
+        try (org.apache.hadoop.hbase.client.Connection tempHConn =
+                ConnectionFactory.createConnection(configuration);
+                RegionLocator regionLocator =
+                        tempHConn.getRegionLocator(TableName.valueOf(tableName))) {
+            numRegions = regionLocator.getStartKeys().length;
```

Comment:
Is there any possibility of getStartKeys returning null?

Reply:
I looked at the implementations, doesn't look like it